tests/saw/spec/handshake/s2n_handshake_io.cry

```diff
@@ -16,14 +16,16 @@ conn_set_handshake_type conn = conn'
                 ,resume_from_cache = conn.resume_from_cache
                 ,server_can_send_ocsp = conn.server_can_send_ocsp
                 ,key_exchange_eph = conn.key_exchange_eph
-                ,client_auth_flag = conn.client_auth_flag}
+                ,client_auth_flag = conn.client_auth_flag
+                ,actual_protocol_version = conn.actual_protocol_version}
```

Comment:
super minor: the final curly bracket ought to be on its own line, for this comma-preceeds-line pattern to do its job.